src/Form/Form.js

```diff
@@ -374,12 +375,17 @@ class Form {
         if (hasIcon) {
             this.addAutofillStyles(input)
             this.addListener(input, 'mousemove', (e) => {
+                if (wasAutofilledByChrome(input)) return
+
                 if (isEventWithinDax(e, e.target)) {
-                    e.target.style.setProperty('cursor', 'pointer', 'important')
+                    addInlineStyles(e.target, {'cursor': 'pointer'})
                 } else {
-                    e.target.style.removeProperty('cursor')
+                    removeInlineStyles(e.target, {'cursor': 'pointer'})
                 }
             })
+            this.addListener(input, 'mouseleave', (e) => {
+                removeInlineStyles(e.target, {'cursor': 'pointer'})
+            })
```

Comment:
In certain cases we could keep the pointer in the field because the cursor moved quickly from Dax to the tooltip, thus the `mousemove` event wouldn't catch that. Then if we dismissed the tooltip we also removed the listeners so we'd have lost our chance. This fixes it and line 199 adds an extra layer to that 🙂.